### PR TITLE
Fix #1694 #1669 #1684: Add contributors and 3rd party libraries

### LINKED ISSUE #1694: Reorder static pages in v2
We have these pages: 
* about
* help
* quickstart

We'll have a page with "Contributors" and "Translators" (see #1684)

I think we can merge some information in all these pages.

### PR DESCRIPTION
**Pull request by nicosomb**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Documentation | no
| Translation   | yes
| Fixed tickets | #1694, #1669, #1684
| License       | MIT

### CODE REVIEW

File: src/Wallabag/CoreBundle/Resources/views/themes/baggy/Static/about.html.twig [hunk 1]
```diff
@@ -42,4 +42,194 @@
...
+        <li>Marmo</li>
+        <li>dsacchet</li>
+        <li>EliasZ</li>
+        <li>skibbipl</li>
+        <li>banux</li>
+        <li>jlnostr</li>
+        <li>aaa2000</li>
+        <li>0wnrepo</li>
+        <li>lologhi</li>
+        <li>0wnrepo</li>
```
Comment (0wnrepo) [inline src/Wallabag/CoreBundle/Resources/views/themes/baggy/Static/about.html.twig]: duplicate of me

File: src/Wallabag/CoreBundle/Resources/views/themes/material/Static/about.html.twig [hunk 2]
```diff
@@ -49,6 +51,196 @@
...
+                            <li>Marmo</li>
+                            <li>dsacchet</li>
+                            <li>EliasZ</li>
+                            <li>skibbipl</li>
+                            <li>banux</li>
+                            <li>jlnostr</li>
+                            <li>aaa2000</li>
+                            <li>0wnrepo</li>
+                            <li>lologhi</li>
+                            <li>0wnrepo</li>
```
Comment (0wnrepo) [inline src/Wallabag/CoreBundle/Resources/views/themes/material/Static/about.html.twig]: same as above

File: src/Wallabag/CoreBundle/Resources/views/themes/baggy/Static/about.html.twig [hunk 3]
```diff
@@ -42,4 +42,194 @@
         <dd>by contributing to the project: <a href="https://github.com/wallabag/wallabag/issues/1254">an issue lists all our needs</a></dd>
         <dd><a href="{{ paypal_url }}">{% trans %}via Paypal{% endtrans %}</a></dd>
     </dl>
+
+    <h2>{% trans %}Contributors{% endtrans %}</h2>
+    <p>{% trans %}Thank you to contributors on wallabag web application{% endtrans %}</p>
+    <ul>
+        <li>nicosomb</li>
+        <li>j0ker</li>
```
Comment (j0k3r) [inline src/Wallabag/CoreBundle/Resources/views/themes/baggy/Static/about.html.twig]: :flushed:

**Comment by j0k3r** [comment]
I think you should better put a lit to the github contributor page instead of listing all users. It'll be a pain to maintain.

**Pull request merged**